DOCS.md

```diff
@@ -27,6 +27,7 @@ The following is a sample Docker configuration in your .drone.yml file:
 ```yaml
 publish:
   docker:
+    privileged: true
```

Comment:
Explicitly setting privileged mode for this plugin isn't required since this plugin is automatically whitelisted to run in privileged mode